README.adoc

```diff
@@ -962,7 +962,7 @@ unless specified otherwise.
 (def *max-size* 10) ; Common Lisp style, global variable
 ----
 
-NOTE: Famously `*clojure-version*` defies this convention, but you should
+NOTE: Famously `\*clojure-version*` defies this convention, but you should
```

Comment:
I think you're supposed to surround this with `+`s, although I always google for this. :D

Reply:
Me too, just googled again, [I think both work](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#escaping-substitutions).

If only there were a style guide for AsciiDoc to help us choose which one! 😄